src/store/reducers/sidebar.js

```diff
@@ -0,0 +1,41 @@
+import { 
+    SIDEBAR_ALL_PROJECT, 
+    SIDEBAR_SAMPLE_PROJECT, 
+    SIDEBAR_UPDATE_PROJECT
+} from '../actionTypes';
+
+const INITIAL_STATE = {
+  allProject: [],
+  sampleProject: [],
+  updateProject: [],
+};
+
+const Sidebar = (state = INITIAL_STATE, action) => {
+  switch (action.type) {
+    case SIDEBAR_ALL_PROJECT:
```

Comment:
Can we do something like this?

case SIDEBAR_ALL_PROJECT:
case SIDEBAR_SAMPLE_PROJECT:
case SIDEBAR_UPDATE_PROJECT:
      return {
        ...state,
        allProject: !!action.data.projects?action.data.projects:[],
	sampleProject: !!action.data.projects?action.data.projects:[],
	updateProject:!!action.data.projects?action.data.projects:[]
      };